bin/main.dart

```diff
@@ -123,6 +236,7 @@ Future<Map<String, String>> _loadStrings(String lang, String dirPath) async {
   return localizedStrings;
 }
 
+// Writing translated files
 _updateContent(Map<String, Map<String, String>> langStrMap, dirPath) {
   Future.forEach(langStrMap.entries, (langStrMapEntry) async {
     String jsonStr = json.encode(langStrMapEntry.value);
```

Comment:
```suggestion
    final jsonStr = json.encode(langStrMapEntry.value);
```

json.encode can also throw an exception. Please handle with try/catch/finally